client/src/components/index.js

```diff
@@ -1,10 +1,46 @@
+require("./style.scss");
 let personsItemComponent = require("./personItem/component"),
     lectureItemComponent = require("./lectureCardItem/component"),
+    fedbackPopUpComponent = require("./fedbackPopUp/component"),
     lectureRowItemComponent = require("./lectureRowItem/component");
 
 module.exports = function(app) {
 
   app.component('lectureRowItem', lectureRowItemComponent);
   app.component('lectureCardItem', lectureItemComponent);
   app.component('personItem', personsItemComponent);
+  app.component('fedbackPopUp', fedbackPopUpComponent);
+
+  const recentTasksController = require("./tasks/recentTasksController.js");
+  app.component('recentTaskItem', {
+    bindings: {
+      task: '='
+    },
+    template: require('./tasks/template.html'),
+    controller: recentTasksController,
+    controllerAs: 'recentTaskItem'
```

Comment:
we can aviod decalring controllerAs for each component. There will be $ctrl by default accessible in template